config/config.go

```diff
@@ -54,7 +61,7 @@ func (s *Service) LoadFile(file string, config interface{}) error {
 		return logger.LogErrorf("unable to load the defaults: %w", err).Err()
 	}
 
-	if err := deflt.Unmarshal(config); err != nil {
+	if err := deflt.Unmarshal(config, errorOnUnusedFields); err != nil {
```

Comment:
Is `UnmarshalExact` has the same effect?

Reply:
Probably? It loads some other functions so I wasn't sure without checking all the properties.